src/zcash/prf.h

```diff
@@ -9,6 +9,14 @@ within the zkSNARK circuit.
 #include "uint256.h"
 #include "uint252.h"
 
+#include <array>
+
```

Comment:
Add comments to separate / distinguish between Sapling and Sprout functions.

Reply:
Added